runner_scripts/get_conan_broadcast_data.py

```diff
@@ -0,0 +1,71 @@
+import argparse
+import os
+import yaml
+
+def get_conan_broadcast_data(args):
+    with open("conandata.yml", "r") as f:
+        conan_data = yaml.safe_load(f)
+    version = conan_data["version"] if args.version == '' else args.version
+    project_name = args.project_name
+
+    if args.release == "true":
+        actual_version = version
+        user = "_"
+        channel = "_"
+        is_release_branch = True
+    else:
+        actual_version = version if "+" in version else f"{version}+{args.sha[:6]}"
+        user = args.user.lower()
+        ref_name = args.base_ref if args.event_name == "pull_request" else args.ref_name
+        if "beta" in version and args.event_name != "pull_request" and  ref_name == '.'.join(version.split('.')[:2]):
+            channel = "stable"
+            is_release_branch = True
+        else:
+            is_release_branch = False
+            if ref_name in ("main", "master"):
+                channel = 'testing'
+            else:
+                channel = "_".join(ref_name.replace("-", "_").split("_")[:2]).lower()
+
+    # %% Set the environment output
+    output_env = os.environ["GITHUB_OUTPUT"]
+    content = ""
+    if os.path.exists(output_env):
+        with open(output_env, "r") as f:
+            content = f.read()
+
+    with open(output_env, "w") as f:
+        f.write(content)
```

Comment:
Why not directly call open with the `"a"` arg?
```suggestion
    with open(output_env, "a") as f:
```